portfolio/src/main/java/com/google/sps/servlets/DeleteDataServlet.java

```diff
@@ -0,0 +1,56 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.Key;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List;
+import com.google.gson.Gson;
+import com.google.appengine.api.datastore.FetchOptions;
+
+/** Servlet that handles comments data */
+@WebServlet("/delete-data")
+public class DeleteDataServlet extends HttpServlet {
+ 
+  private DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+  
+  @Override
+  public void init() {
+    datastore = DatastoreServiceFactory.getDatastoreService();
+  }
+  
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query commentKeysQuery = new Query("Comment").setKeysOnly();
+    PreparedQuery commentKeysResults =  datastore.prepare(commentKeysQuery);
+    for(Entity commentKey : commentKeysResults.asIterable()){
+      datastore.delete(commentKey.getKey());
```

Comment:
Is there a "bulk deletion" API available in datastore? It would almost certainly be more efficient to collect the keys then delete them all simultaneously.

Reply:
https://cloud.google.com/datastore/docs/bulk-delete
good point, seems like there is some setup involved, will look into tomorrow

Reply:
SGTM

Reply:
https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/master/src/main/java/com/google/cloud/teleport/templates/DatastoreToDatastoreDelete.java
& 
https://cloud.google.com/dataflow/docs/guides/specifying-exec-params#configuring-pipelineoptions-for-local-execution
seem to be good documentation for this, but there are a lot of gaps right now in my understanding, which l'm trying to get cleared up in the Technical Support Chat.
For now l'll merge this PR and open another one once l figure out how to use the Dataflow API